src/Microsoft.Health.Fhir.Shared.Api/Registration/FhirServerServiceCollectionExtensions.cs

```diff
@@ -93,16 +94,17 @@ public static class FhirServerServiceCollectionExtensions
         }
 
         /// <summary>
-        /// Adds the export worker background service.
+        /// Adds background worker services.
         /// </summary>
         /// <param name="fhirServerBuilder">The FHIR server builder.</param>
         /// <returns>The builder.</returns>
-        public static IFhirServerBuilder AddExportWorker(
+        public static IFhirServerBuilder AddBackgroundWorkers(
             this IFhirServerBuilder fhirServerBuilder)
         {
             EnsureArg.IsNotNull(fhirServerBuilder, nameof(fhirServerBuilder));
 
             fhirServerBuilder.Services.AddHostedService<ExportJobWorkerBackgroundService>();
+            fhirServerBuilder.Services.AddHostedService<ReindexJobWorkerBackgroundService>();
```

Comment:
Shouldn't we also register the `ReindexJobConfiguration` as an Option so that the IoC container knows about it (like the others in `AddFhirServer`)?

Reply:
Yes!  good catch, thank you.